api/edge_api/identities/serializers.py

```diff
@@ -20,6 +22,28 @@
 engine_multi_fs_value_schema = MultivariateFeatureStateValueSchema()
 
 
+class EdgeIdentitySerializer(serializers.ModelSerializer):
```

Comment:
I think this has just moved so apologies for not adding the comment before but I don't think this should be a ModelSerializer since we're not working with a django model here.

Reply:
Yes, agreed but if we make it `serializers.Serializer` then we have to manually add all of those fields to the serializer which is kinda redundant?

Reply:
And add create/update methods since we are using model viewset

Reply:
Since `environment` is a read only field, I don't see that there's any validation that we need here? The only validation is that the identifier is a valid string and maybe we also want to ensure that it's fewer than 2000 chars (although I guess this is less important with edge identities)?

Also, the `create` / `update` methods are only called by `save` so by overriding save, we don't need them.

Reply:
> although I guess this is less important with edge identities?

Wait, why?

Reply:
Well, because we're not limited on the size of the field in the database (since these will only be stored in dynamo and never reach postgres). We should still enforce the limit, but it's not essential.

Reply:
Ah, okay